scripts/post_install_script.sh

```diff
@@ -103,59 +105,90 @@ case "${cfn_node_type}" in
         #NOTE: as of parallelcluster v2.8.0, SLURM is built with PMI3
         cat << EOF > ${spack_install_path}/etc/spack/packages.yaml
 packages:
```

Comment:
can we separate this out into it's own file?

Reply:
We can embed this as a separate file, but it will require hardcoding an S3 bucket/URL to get the file. Will come back to this.

Reply:
Done